cask.el

```diff
@@ -402,8 +405,7 @@ SCOPE may be nil or 'development."
          (cl-destructuring-bind (_ &rest patterns) form
            (setf (cask-bundle-patterns bundle) patterns)))
         (development
-         (cl-destructuring-bind (_ . body) form
-           (cask--eval bundle body 'development)))
+         (cask--eval bundle (cdr form) 'development))
```

Comment:
Not related to this, right? In such case, please remove and make a separate pull request. We want to change as little as possible at a time.